packages/storybook/.storybook/preview.js

```diff
@@ -1,6 +1,6 @@
-import '@department-of-veterans-affairs/formation/dist/formation.min.css';
+// import '@department-of-veterans-affairs/formation/dist/formation.min.css';
 import './style.scss';
-import '@department-of-veterans-affairs/formation/dist/formation';
+// import '@department-of-veterans-affairs/formation/dist/formation';
```

Comment:
For completeness, it might be a good idea to test all of this out with Formation back in place. We have already merged the Storybook/Formation/v2 overrides into the main branch so if this branch is up to date, removing this shouldn't be necessary anymore: https://github.com/department-of-veterans-affairs/component-library/pull/577

Reply:
I agree